MVP Module/Source/Feed UI/Controllers/FeedViewController.swift

```diff
@@ -26,17 +27,21 @@ public final class FeedViewController: UITableViewController, UITableViewDataSou
 	}
 
 	func display(_ viewModel: FeedLoadingViewModel) {
-		if viewModel.isLoading {
-			refreshControl?.beginRefreshing()
-		} else {
-			refreshControl?.endRefreshing()
-		}
+		viewModel.isLoading ? refreshControl?.beginRefreshing() : refreshControl?.endRefreshing()
 	}
 
 	func display(_ cellControllers: [FeedImageCellController]) {
 		tableModel = cellControllers
 	}
 
+	func hideMessage() {
+		errorView.hideMessage()
+	}
+
+	func displayError(_ error: String) {
+		errorView.show(message: error)
+	}
```

Comment:
We can make the `FeedViewController` conform to `FeedErrorView` (the same way it conforms to `FeedLoadingView`, and implement the display method:
```suggestion
	func display(_ viewModel: FeedErrorViewModel) {
		if let errorMessage = viewModel.errorMessage {
			errorView.show(message: errorMessage)
		} else {
			errorView.hideMessage()
		}
	}
```
To do that, remember to make `FeedViewController` conform to error view, either in an extension or in the class declaration:
```swift
public final class FeedViewController: UITableViewController,
	UITableViewDataSourcePrefetching, FeedLoadingView, FeedErrorView {
```

Reply:
done